sympy/physics/control/lti.py

```diff
@@ -1739,6 +1878,19 @@ def doit(self, cancel=False, **kwargs):
         TransferFunctionMatrix(((TransferFunction(2*(-p + s)*(s**3 - 2)*(a*p**2 + b*s)*(s**4 + 5*s + 6), (-p + s)**2*(s**4 + 5*s + 6)**2, s), TransferFunction((-p + s)**2*(s**3 - 2)*(a*p**2 + b*s) + (-p + s)*(a*p**2 + b*s)**2*(s**4 + 5*s + 6), (-p + s)**3*(s**4 + 5*s + 6), s)), (TransferFunction((-p + s)*(s**3 - 2)**2*(s**4 + 5*s + 6) + (s**3 - 2)*(a*p**2 + b*s)*(s**4 + 5*s + 6)**2, (-p + s)*(s**4 + 5*s + 6)**3, s), TransferFunction(2*(s**3 - 2)*(a*p**2 + b*s), (-p + s)*(s**4 + 5*s + 6), s))))
 
         """
+        if self._is_series_StateSpace:
+            # Return the equivalent StateSpace model
+            res = self.args[0]
+            if not isinstance(res, StateSpace):
+                res = res.doit().rewrite(StateSpace)
+            for arg in self.args[1:]:
+                if not isinstance(arg, StateSpace):
+                    arg = arg.doit().rewrite(StateSpace)
+                else:
+                    arg = arg.doit()
+                res = arg * res
```

Comment:
I've noticed an problem with the previous code so I've changed it from 
`res *= arg` to `res = arg * res`.
I've refereed to this :
 
![image](https://github.com/sympy/sympy/assets/140839576/7f59d00e-db9a-4e38-ae36-86ffbfad3194)

Source : https://assets.press.princeton.edu/chapters/s9012.pdf